certbot-apache/certbot_apache/assertions.py

```diff
@@ -51,10 +55,59 @@ def assertEqualDirective(first, second):
     assert isinstance(second, interfaces.DirectiveNode)
     _assertEqualDirectiveComponents(first, second)
 
+def assertEqualBlock(first, second):
+    """ Equality assertion for BlockNode """
+
+    # first was checked in the assertEqual method
+    assert isinstance(first, interfaces.BlockNode)
+    assert isinstance(second, interfaces.BlockNode)
+    _assertEqualDirectiveComponents(first, second)
+    # Children cannot be asserted, because Augeas implementation will not
```

Comment:
if we're not asserting about children, this is basically the same check as `assertEqualDirective`; is there any benefit to having this around as well?

Reply:
Only the actual interface implementation assertion, it's now moved to `assertEqual`